src/Chainweb/Mempool/RestAPI/Server.hs

```diff
@@ -54,11 +53,9 @@ lookupHandler mempool txs = handleErrs (liftIO look)
     look = V.toList <$> mempoolLookup mempool txV
 
 
+-- TODO: should we re-implement get block for remote?
 getBlockHandler :: Show t => MempoolBackend t -> Maybe GasLimit -> Handler [t]
```

Comment:
What would be the reason for the function? Remove it?

Reply:
It's not used at the moment so there's no downside in deleting it